README.md

```diff
@@ -75,9 +75,13 @@
       - [`editField(id[, opts])` : [Field](#field)](#editfieldid-opts--fieldfield)
       - [`deleteField(id)` : void](#deletefieldid--void)
       - [`changeFieldId (currentId, newId)` : void](#changefieldid-currentid-newid--void)
-      - [`changeEditorInterface (fieldId, widgetId[, settings])` : void](#changeeditorinterface-fieldid-widgetid-settings--void)
-      - [`resetEditorInterface (fieldId)` : void](#reseteditorinterface-fieldid--void)
-      - [`copyEditorInterface (sourceFieldId, destinationFieldId)` : void](#copyeditorinterface-sourcefieldid-destinationfieldid--void)
+      - [`changeFieldControl (fieldId, widgetId[, settings, widgetNamespace])` : void](#changefieldcontrol-fieldid-widgetid-settings-widgetnamespace--void)
+      - [`resetFieldControl (fieldId)` : void](#resetfieldcontrol-fieldid--void)
+      - [`copyFieldControl (sourceFieldId, destinationFieldId)` : void](#copyfieldcontrol-sourcefieldid-destinationfieldid--void)
+      - [`addSidebarWidget (widgetId[, widgetNamespace, insertBeforeWidgetId, settings])` : void](#addsidebarwidget-widgetid-widgetnamespace-insertbeforewidgetid-settings--void)
+      - [`updateSidebarWidget (widgetId, settings)` : void](#updatesidebarwidget-widgetid-settings--void)
+      - [`removeSidebarWidget (widgetId)` : void](#removesidebarwidget-widgetid--void)
```

Comment:
For this one we need the namespace for sure. There may be a widget with the same ID in multiple namespaces.

Reply:
Extension ID's are some form of UUIDs or? And builtin stuff is a descriptive string. How are we going to make them collide?

Reply:
You can PUT to `/extensions/:id` to set your own ID if the entity doesn't exist yet.

It's actually the default behavior of the Contentful CLI (uses `id` property from `extension.json`)

Reply:
I'll change it